spec/fixtures/apply/basic/plans/inventory_2_lookup.pp

```diff
@@ -0,0 +1,11 @@
+plan basic::inventory_2_lookup(TargetSpec $nodes) {
+  # TODO: Provide inventory data to bolt_catalog to support get_targets in apply
+  return apply($nodes) {
+    # get_targets should return empty list for 'all'
+  	$t_all = get_targets('all')
```

Comment:
This indentation seems weird

Reply:
I had tab size set to 4 for some reason in my text editor. Updated.